tests/cases/compiler/pathMappingBasedModuleResolution8_node.ts

```diff
@@ -0,0 +1,23 @@
+// @moduleResolution: node
+// @module: commonjs
+// @traceResolution: true
+
+// @filename: c:/root/tsconfig.json
+{
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "@speedy/*/testing": [
+               "*/dist/index.ts"
+            ]
+        }
+    }
+}
+
+// @filename: c:/root/index.ts
+import {x} from "@speedy/folder1/testing"
+declare function use(a: any): void;

```

Comment:
Are these two lines necessary? `--noUnusedLocals` is disabled by default.

Reply:
In that case it's safe to remove